pulsar-function-go/pf/instance.go

```diff
@@ -21,12 +21,13 @@ package pf
 
 import (
 	"context"
+	"github.com/golang/protobuf/ptypes/empty"
 	"math"
 	"time"
 
 	"github.com/apache/pulsar/pulsar-client-go/pulsar"
 	log "github.com/apache/pulsar/pulsar-function-go/logutil"
-	"github.com/apache/pulsar/pulsar-function-go/pb"
+	pb "github.com/apache/pulsar/pulsar-function-go/pb"
```

Comment:
```suggestion
	"github.com/apache/pulsar/pulsar-function-go/pb"
```

Reply:
When I make this change, I can't get it to build. I get unresolved variables, even if I remove pb from the references in the code.